### DISCUSSION TITLE
Feature request: keep responsive values (e.g. padding) unlinked and don't reset to 0

### DISCUSSION THREAD

**Discussion by m1ga** [elementor/elementor#25830] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

As it is not handled as a [bug](https://github.com/elementor/elementor/issues/16019) I try it as a features request.

Easy setup: you create a page in Elementor and set a padding/margin with unlinked values. Then you want to set the margin in a responsive breakpoint to something else: the values are not unlinked anymore and if you unlink it it will be 0.

**Video:**

https://github.com/elementor/elementor/assets/4334997/a9056e7a-3106-435e-a599-1da489e2c3cb

**Steps:**
* set a padding/margin and unlink the values.
*  set different values for each field
* switch a lower breakpoint
* link status is not correct
* when you unlink it again all fields will be zero and not the numbers that were there before.

### Describe the solution you'd like.

* Keep the link/unlinked status
* don't reset it to 0 if you unlink it and it has values from the breakpoint above

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by thijmenvdijk** [comment]
I would love this indeed. It is really frustating.
